cesm/driver/esm_time_mod.F90

```diff
@@ -141,6 +141,7 @@ subroutine esm_time_clockinit(ensemble_driver, instance_driver, logunit, maintas
                 inquire( file=trim(restart_pfile), exist=exists)
                 if (.not. exists) then
                    rc = ESMF_FAILURE
+                   write(logunit,*) " drv_restart_pointer file does NOT exist, correct this and rerun = "//trim(restart_pfile)
```

Comment:
I'm curious if we have a general rule in CMEPS around what gets written to logunit, what gets written to the ESMF log files, and what gets written to both. I'm okay with the additions you made here to logunit, but I'd like to know the general rule (if there is one) to guide this and future logging (since, to me, it feels possibly unnecessary to write effectively the same message to multiple log files).

Reply:
Oh, and I see that this follows from discussion in #527, which I missed in my inbox overload last week. I'll pop over there to chime in.